dxr/templates/layout.html

```diff
@@ -49,20 +49,25 @@
                           </button>
                         </section>
                         <div class="sf-select-options sf-modal" aria-expanded="false">
-                          <ul class="selector-options" tabindex="-1">
+                          <table class="selector-options" tabindex="-1">
                             {% for filter in filters %}
-                              <li>
-                                <a href="javascript:void(0)" data-value="{{ filter.name }}:">
-                                  <span class="selector-option-label">
-                                    {{ filter.name }}
-                                  </span>
-                                  <span class="selector-option-description">
-                                    {{ filter.description }}
-                                  </span>
-                                </a>
-                              </li>
+                              <tr data-value="{{ filter.name }}:">
+                                <td class="selector-option-label">
```

Comment:
Let's make sure this cell gets valigned top (the CSS equivalent, that is), to match the description and make it clear which row the label goes with. See 
![screen shot 2016-07-25 at 11 01 49](https://cloud.githubusercontent.com/assets/295816/17105968/628c8220-5257-11e6-898a-efe9f303f21b.png).